src/PHPSemVerCheckerGit/Console/Command/SuggestCommand.php

```diff
@@ -50,8 +50,8 @@ private function getRepository($directory)
     }
 
 	/**
-	 * @param \Symfony\Component\Console\Input\InputInterface   $input
-	 * @param \Symfony\Component\Console\Output\OutputInterface $output
+	 * @param InputInterface   $input
```

Comment:
Be careful, we still want the FQCN (fully qualified class name).

Reply:
having FQN in PHPdoc only looks inconsistent to me when they aren't also used elsewhere, Will fix those.